.travis.yml

```diff
@@ -2,21 +2,20 @@ language: node_js
 node_js:
     - "0.12"
     - "0.10"
-#   - "0.8"   # no longer supported by iconv
-#   - "0.6"   # no longer supported by async
```

Comment:
These comments, telling **why** we don't support node 0.8 and 0.6 tend to be helpful, which is why we leave them in there.